proxies/live/apiproxy/targets/ers-target.xml

```diff
@@ -15,20 +15,21 @@
             </Step>
             <Condition>(fault.name Matches "SpikeArrestViolation")</Condition>
         </FaultRule>
+        <FaultRule name="quota_fault">
+            <Step>
+                <Name>AssignMessage.QuotaErrorResponse</Name>
+            </Step>
+            <Condition>(fault.name Matches "QuotaViolation")</Condition>
```

Comment:
There will be a flow variable ratelimit.SpikeArrest.Referrals.failed, also for the shared flow, that would be better to use here.